tests/models/test_models.py

```diff
@@ -203,7 +203,7 @@ def test_raster_schema(
     @pytest.mark.parametrize("model", [ShapesModel])
     @pytest.mark.parametrize("path", [POLYGON_PATH, MULTIPOLYGON_PATH, POINT_PATH])
     def test_shapes_model(self, model: ShapesModel, path: Path) -> None:
-        radius = RNG.normal(size=(2,)) if path.name == "points.json" else None
+        radius = np.abs(RNG.normal(size=(2,))) if path.name == "points.json" else None
```

Comment:
if we are going to use an RNG, we should set the seed. otherwise, test failures might be probabilistic.

Reply:
true, added